docs/features/README.md

```diff
@@ -1,2 +1,3 @@
 # Features
 * [ ] [1. Hello World](./features/1-hello-world.md)
+* [ ] [2. Loops](./features/2-loops.md)
```

Comment:
Going forward, the next branch to check out can be found here -
https://github.com/Make-Me-Coder/Javascript/wiki

This is a list of branches in sorted order.